src/main/java/org/example/Ships/Shots.java

```diff
@@ -0,0 +1,24 @@
+package org.example.Ships;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+public class Shots {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id_shot;
```

Comment:
to samo :D (jak chcesz zmienic nazwe wyswietlana w bazie danych jest od tego annotacja @Column :)